src/views/Home.vue

```diff
@@ -0,0 +1,4 @@
+<template>
+this is home truly, <br>
+beautiful home page
```

Comment:
brainwashed song

In all seriousness please wrap this in a `<p>` since it's a paragraph